crates/types/src/config/worker.rs

```diff
@@ -594,3 +607,34 @@ impl SnapshotsOptions {
         super::data_dir("db-snapshots").join(partition_id.to_string())
     }
 }
+
+/// # Throttling options
+///
+/// Throttling options per invoker.
+#[derive(Debug, Clone, Serialize, Deserialize, derive_builder::Builder)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[cfg_attr(feature = "schemars", schemars(rename = "ThrottlingOptions", default))]
+#[builder(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct ThrottlingOptions {
+    /// # Burst capacity
+    ///
+    /// The maximum number of tokens the bucket can hold.
+    /// Default: 1M
+    pub burst: NonZeroU32,
+
+    /// # Replenishment rate
+    ///
+    /// The rate in seconds at which the tokens are replenished.
```

Comment:
This can be on a follow-up PR but it'd be nice if:
- The rate can be parsed from a string `5/h` and if literal number then it's per second by default.
- If burst is optional, by default it inheriting the rate's value.

Reply:
Will do